CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing to Microsoft.Data.SqlClient
+
+General contribution guidance is included in this document. Additional guidance is defined in the documents linked below.
+
+- [Copyright](copyright.md) describes the licensing practices for the project.
+- [Contribution Workflow](contributing-workflow.md) describes the workflow that the team uses for considering and accepting changes.
+
+## Up for Grabs
+
+The team marks the most straightforward issues as "up for grabs". This set of issues is the place to start if you are interested in contributing but new to the codebase.
+
+- [dotnet/sqlclient - "up for grabs"](https://github.com/dotnet/sqlclient/labels/Up-for-Grabs)
+
+## Contribution "Bar"
+
+Project maintainers will merge changes that improve the product significantly and broadly and that align with the [Microsoft.Data.SqlClient roadmap](https://github.com/dotnet/sqlclient/blob/master/roadmap.md).
+
+Contributions must also satisfy the other published guidelines defined in this document.
+
+## DOs and DON'Ts
+
+Please do:
+- **DO** report each issue as a new issue (but check first if it's already been reported)
+- **DO** respect Issue Templates and provide detailed information. It will make the process to reproduce the issue and provide a fix faster.
+- **DO** provide a minimal repro app demonstrating the problem in isolation will greatly speed up the process of identifying and fixing problems.
+- **DO** follow our [coding style](coding-style.md) (C# code-specific) when working on a Pull Request.
+- **DO** give priority to the current style of the project or file you're changing even if it diverges from the general guidelines.
+- **DO** consider cross-platform compatibility and supportability for all supported SQL and Azure Servers and client configurations.
+- **DO** include tests when adding new features. When fixing bugs, start with adding a test that highlights how the current behavior is broken.
+- **DO** consider concurrency when writing tests. Out pipelines run builds and tests in parallel using the same client and server configurations (in an isolated mode). E.g. Consider using dynamic table/database object names instead of hardcoded values (Use existing tests for reference).
+- **DO** keep the discussions focused. When a new or related topic comes up it's often better to create new issue than to side track the discussion.
+- **DO** blog and tweet (or whatever) about your contributions, frequently!
```

Comment:
Why is this suggested? I don't have a blog or twitter or anything like that.

Reply:
I brought in some stuff from corefx repro, it came with it :)
Feel free to ignore if it's not applicable to you. I don't see any harm in promoting contributions!

Reply:
No, it just seems a bit weird and I don't remember seeing it in the corefx version. It isn't a problem.